configure.ac

```diff
@@ -951,6 +951,30 @@
         LIBNET_FAIL_WARN($libnet_dir)
     fi
 
+  # libbsd
+     AC_ARG_WITH(libbsd_includes,
+            [  --with-libbsd-includes=DIR  libbsd include directory],
+            [with_libbsd_includes="$withval"],[with_libbsd_includes=no])
+
+    if test "$with_libbsd_includes" != "no"; then
+        CPPFLAGS="${CPPFLAGS} -I${with_libbsd_includes}"
+    fi
+
+    LIBBSD=""
+    AC_CHECK_HEADER(bsd/sys/tree.h,, LIBBSD="no")
+
+    if test "$LIBBSD" = "no"; then
+        echo
+        echo "   ERROR!  libbsd-dev files not found, go get them"
+        echo
+        echo "   Ubuntu: apt-get install libbsd-dev"
+        echo "   Fedora: yum install libbsd-devel"
+        echo
+        exit 1
+    fi   
+     
+    AC_DEFINE([HAVE_SYS_TREE_H],[1],[bsd/sys/tree.h available])
+    
```

Comment:
I wonder if we should just include tree.h with Suricata like we do queue.h...  Or if depending on libbsd, we could drop queue.h and use the one from libbsd.